.github/workflows/test_julia_targets.yml

```diff
@@ -0,0 +1,73 @@
+name: julia test
+
+# Controls when the action will run.
+on:
+  # push:
+  #   branches:
+  #     - main
+  # pull_request:
+  #   branches:
+  #     - main
+  # # run once a week - - github servers often get a lot of requests on the hour and may fail silently 
+  # schedule:
+  #   - cron: "21 0 * * *"
+  workflow_dispatch:
+jobs:
+  targets_julia:
+    runs-on: ubuntu-latest
+    container:
+      image: rocker/verse:4.5.0
+    steps:
+      - name: update permissions for container based workflows
+        run: git config --global --add safe.directory /__w/virion/virion
+      - uses: actions/checkout@v4 # https://github.com/actions/checkout
+      - name: Install system dependencies
+        run: |
+          sh sys_deps/sys_deps.sh
+      - name: Install packages from renv.lock (with cache)
+        if: ${{ !env.ACT }}  ## Doesn't work locally with ACT
+        uses: r-lib/actions/setup-renv@v2
+        with:
+          cache-version: 4
+      - name: Install packages from renv.lock (local, no cache)
+        if: ${{ env.ACT }}  ## Only locally with ACT, use `act -r` to reuse containers, effectively caching locally
+        run: |
+          renv::restore()
+        shell: Rscript {0}
+      - name: Install Juliaup
+        uses: julia-actions/install-juliaup@v2
+        with:
+          channel: '1.9'
```

Comment:
should be changed to `lts`

Reply:
do we want to keep this on julia version 1.9 for some reason instead of >1.10/11?